boost-maven/boost-boms/boost-ee7-apis-bom/pom.xml

```diff
@@ -36,6 +36,16 @@
                 <artifactId>javax.transaction-api</artifactId>
                 <version>1.2</version>
             </dependency>
+            <dependency>
```

Comment:
This PR, hitting this many files, is probably a good time to try to format our XML more consistently.   Can you suggest a format, and  apply consistently for the changed parts?   I'd suggest four spaces..that way we can do a simple grep for tab as a check at some point.